tutorials/diagnostics.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>You really do two things here: Access the Jacobian and plot the sparsity pattern. In a tutorial you want to be explicit on such nuances.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/813/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>